### ISSUE TITLE
Connection and _connectionSchema objects are equal.

### ISSUE THREAD

**Issue by burnout171** [OpenNetworkingFoundation/Snowmass-ONFOpenTransport#110]
During analyzing TapiConnectivity.swagger file we observed that Connection and _connectionSchema  have equal fields.
Connection
```
  Connection:
    description: >-
      The ForwardingConstruct (FC) object class models enabled potential for
      forwarding between two or more LTPs and like the LTP supports any
      transport protocol including all circuit and packet forms.

      At the lowest level of recursion, a FC represents a cross-connection
      within an NE.
    allOf:
      - $ref: '#/definitions/ResourceSpec'
      - properties:
          direction:
            enum:
              - BIDIRECTIONAL
              - UNIDIRECTIONAL
              - UNDEFINED_OR_UNKNOWN
            type: string
            description: none
          layerProtocolName:
            enum:
              - OCH
              - ODU
              - ETH
              - MPLS_TP
            type: string
            description: none
          _route:
            items:
              $ref: '#/definitions/Route'
            type: array
            description: none
            x-key: localId
          _connectionPort:
            items:
              $ref: '#/definitions/ConnectionPort'
            type: array
            description: none
            x-key: localId
          _state:
            description: none
            $ref: '#/definitions/OperationalStatePac'
          _node:
            type: string
            description: none
            x-path: '/Tapi:Context/Tapi:_topology/TapiTopology:_node/TapiTopology:uuid'
```

_connectionSchema
```
  _connectionSchema:
    allOf:
      - $ref: '#/definitions/ResourceSpec'
      - properties:
          direction:
            enum:
              - BIDIRECTIONAL
              - UNIDIRECTIONAL
              - UNDEFINED_OR_UNKNOWN
            type: string
            description: none
          layerProtocolName:
            enum:
              - OCH
              - ODU
              - ETH
              - MPLS_TP
            type: string
            description: none
          _route:
            items:
              $ref: '#/definitions/Route'
            type: array
            description: none
            x-key: localId
          _connectionPort:
            items:
              $ref: '#/definitions/ConnectionPort'
            type: array
            description: none
            x-key: localId
          _state:
            description: none
            $ref: '#/definitions/OperationalStatePac'
          _node:
            type: string
            description: none
            x-path: '/Tapi:Context/Tapi:_topology/TapiTopology:_node/TapiTopology:uuid'
```

What do you think about to unite these 2 object? Current definition may lead to inconsistency issues in the future.

**Comment by rvilalta** [comment]
You are right. Both objects should be the same.

**Comment by rvilalta** [comment]
Fixed in #117
